[AndroidManifest] Save/Restore/Diff Merged Manifest

ParaskP7 opened the pull request:
Project Thread: paaHJt-7Tl-p2
Related: #164 (`new workflow`)

---

## Description

`TODO`

---

- [ ] I have considered if this change warrants release notes and have added them to the appropriate section in the `CHANGELOG.md` if necessary.

ParaskP7 committed:
Android: Add save android merged manifest script

dangermattic commented:
<!--
  0 Errors
  1 Warning: Please add an entry in the <co...
  1 Message: This PR is still a Draft: some...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please add an entry in the <code>CHANGELOG.md</code> file to describe the changes made by this PR</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

ParaskP7 committed:
Android: Add restore android merged manifest script

ParaskP7 committed:
Android: Add comment with manifest diff script

ParaskP7 committed:
Android: Enhance manifest save, restore and diff with build variant

ParaskP7 committed:
Android: Add module suffix to manifest diff id to enable multi comments

This is done so that when multiple modules exist, and more than one
module has a manifest diff, that the previous comment is not overridden
by the newest comment, and thus enabling multiple manifest diff
comments.

ParaskP7 commented:
Closing this in favor of #164 .
